src/component/admin/apiToken/ApiTokenForm/SelectProjectInput/SelectProjectInput.tsx

```diff
@@ -79,13 +83,20 @@ export const SelectProjectInput: VFC<ISelectProjectInputProps> = ({
 
     const renderGroup = ({ key, children }: AutocompleteRenderGroupParams) => (
         <Fragment key={key}>
-            <SelectAllButton
-                isAllSelected={isAllSelected}
-                onClick={() => {
-                    setProjects(
-                        isAllSelected ? [] : options.map(({ value }) => value)
-                    );
-                }}
+            <ConditionallyRender
+                condition={options.length > 2}
+                show={
+                    <SelectAllButton
+                        isAllSelected={isAllSelected}
+                        onClick={() => {
```

Comment:
Could extract this into a `const onSelectAllClick = ...`, just for readability.

Reply:
refactored 🙂